app/presenters/auction_presenter.rb

```diff
@@ -132,6 +134,15 @@ def human_start_time
     end
   end
 
+  def url
+    root_url = if Rails.env.development? || Rails.env.test?
+                 ENV['ROOT_URL']
```

Comment:
Code Climate/Rubucop demanded it. Bow before your static code analysis overlords!